src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid var(--card-border);
+
+  &__logo {
+    box-sizing: border-box;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+
+    &-image {
+      width: 100%;
+      height: 100%;
+    }
+  }
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 16px;
+    width: 166px;
+    height: 40px;
+    background-color: var(--link-color);
+    text-transform: uppercase;
+    text-decoration: none;
+    color: var(--link-text-color);
+    border-radius: 5px;
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-size: 14px;
```

Comment:
You can add font styles to your html and wil not need to repeat it to every selector